microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -589,7 +589,7 @@ Microsoft Edge asks you if you want to store your payment information if payment
 
 Microsoft Edge also supports saving your payment information to your Microsoft account if you are signed in and syncing, which makes the information available across devices. To save payment information to your Microsoft account, credit card verification (CVV) may be required, depending on your current region. CVV is only used for authorization and will not be stored by Microsoft. 
 
-Microsoft Edge supports the PaymentRequest API. The API lets you pay for purchases with payment information you previously saved using autofill. The PaymentRequest API allows the merchant to request the following information: credit card number, credit card expiration, full name, billing address, email address, phone number, and shipping address. The API tells the merchant that you have credit card information saved, but does not share any information with the merchant unless you allow it. To turn off the Payments feature, go to `edge://settings/privacy`.
+Microsoft Edge supports the PaymentRequest API. The API lets you pay for purchases with payment information you previously saved using autofill. The PaymentRequest API allows the merchant to request the following information: credit card number, credit card expiration, full name, billing address, email address, phone number, and shipping address. The API tells the merchant that you have credit card information saved, but does not share any information with the merchant unless you allow it. To turn off the Payments feature, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->
```

Comment:
I don't believe I've seen this in Settings before. 
I dont see it on either Services or Privacy and security. 

I believe this is supposed to be **edge://wallet/paymentMethods**

Reply:
and then select p.c.s. - gives: 
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/abc046e7-6527-483d-9e7e-427dd8ebd115)

Reply:
```
To turn off the Payments feature, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings**, and then in the **Search settings** text box, enter **wallet**.  
Click the **Open Wallet** button, and then select **Settings** on the left.
```